File: lifecycle-sample/src/main/java/com/sun/android/lifecycle/sample/TestActivity.java
package com.sun.android.lifecycle.sample;

import android.os.Bundle;

import java.security.interfaces.DSAKey;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.sun.android.util.Logs;

/**
 * @author tianxiaolong
 * time：2020/1/17 17:37
 * description：
 */
public class TestActivity extends AppCompatActivity {
	@Override
	protected void onCreate(@Nullable Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		Logs.INSTANCE.I("Test onCreate");
	}


	@Override
	protected void onResume() {
		super.onResume();
		Logs.INSTANCE.I("Test onResume");
	}

	@Override
	protected void onDestroy() {
		super.onDestroy();

	}
}

File: util-lib/src/main/java/com/sun/android/util/TestActivity.kt
package com.sun.android.util

import android.app.Activity
import android.os.Bundle

/**
 *
 * @author tianxiaolong
 * time：2020/4/20 10:33 PM
 * description：
 */
class TestActivity :Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startUiActivity<TestActivity>{
            putExtra("s","s")
            putExtra("a","a")
            putExtra("b","b")
        }
    }
}